modules/local/orthofinder_prep.nf

```diff
@@ -0,0 +1,58 @@
+process ORTHOFINDER_PREP {
+    tag "Prepping data for OrthoFinder"
```

Comment:
Okay - maybe tomorrow let's decide which modules it makes sense to use tags for. Maybe just the modules where I'm dealing with individual species?

I honestly just started using tags this way since I just saw it was printing to terminal.